client/src/app/shared/forms/form-validators/video-channel-validators.service.ts

```diff
@@ -14,28 +14,28 @@ export class VideoChannelValidatorsService {
     this.VIDEO_CHANNEL_NAME = {
       VALIDATORS: [
         Validators.required,
-        Validators.minLength(3),
-        Validators.maxLength(20),
-        Validators.pattern(/^[a-z0-9._]+$/)
+        Validators.minLength(1),
+        Validators.maxLength(50),
+        Validators.pattern(/^[a-zA-Z0-9][a-zA-Z0-9-._]*$/)
```

Comment:
same here (lowercase)

Reply:
ok sounds good

Reply:
I'll work on the `server/tests/api/check-params/video-channels.ts` right now

Reply:
The video channel name tests seem to be fine. We could add more test cases to check username `...` or channel name `...` etc that don't meet the criteria of the regex, but the one check with a space in the name 'super name' already covers one use case that should fail, maybe we should add more failing use cases?